KronorComponents/Common/EmbeddedPaymentViewModel.swift

```diff
@@ -15,6 +15,7 @@ enum SupportedEmbeddedMethod {
     case mobilePay
     case creditCard
     case vipps
+    case payPal
     case fallback(name: String)
     
     func getName() -> String {
```

Comment:
Suggestion - `getName()` could be shortened by saying 
```
    func getName() -> String {
        switch self {
        case .fallback(let name):
            return name
        default: 
            return String(describing: self)
        }
    }
```

Reply:
I'll create a PR for this later. Thanks!